lib/gollum-lib/sanitization.rb

```diff
@@ -1,378 +1,52 @@
-# ~*~ encoding: utf-8 ~*~
+::Loofah::HTML5::SafeList::ACCEPTABLE_PROTOCOLS.add('apt')
```

Comment:
We have a test that checks whether links can have the apt scheme. Also fine with removing that test.